Changelog.md

```diff
@@ -8,6 +8,7 @@
 - Fix bug where grader distribution graph displays graders multiple times (#6950)
 - Fixed bug where TA Summary table failed to display members (#6949)
 - Make display of group member information consistent across submissions and summary table (#6917)
+- Add new routes for `update`, `show`, and `index` actions of the Sections API Controller (#6955)
```

Comment:
Please do a merge from master, and then make sure this changelog entry is under `[unreleased]`